src/appkit.cs

```diff
@@ -28450,5 +28426,88 @@ interface NSImageSymbolConfiguration : NSCopying, NSSecureCoding
 		[Mac (12,0)]
 		[Export ("configurationByApplyingConfiguration:")]
 		NSImageSymbolConfiguration Create (NSImageSymbolConfiguration configuration);
+
+		[Mac (13,0)]
+		[Static]
+		[Export ("configurationPreferringMonochrome")]
+		NSImageSymbolConfiguration ConfigurationPreferringMonochrome ();
+
+		[Mac (13,0)]
+		[Static]
+		[Export ("configurationPreferringHierarchical")]
+		NSImageSymbolConfiguration ConfigurationPreferringHierarchical ();
+	}
+
+	[NoMacCatalyst, Mac (13,0)]
+	[BaseType (typeof(NSControl))]
+	interface NSComboButton
+	{
+		[DesignatedInitializer]
+		[Export ("initWithFrame:")]
+		NativeHandle Constructor (CGRect frameRect);
+
+		[Static]
+		[Export ("comboButtonWithTitle:menu:target:action:")]
+		NSComboButton CreateWithTitle (string title, [NullAllowed] NSMenu menu, [NullAllowed] NSObject target, [NullAllowed] Selector action);
+
+		[Static]
+		[Export ("comboButtonWithImage:menu:target:action:")]
+		NSComboButton CreateWithImage (NSImage image, [NullAllowed] NSMenu menu, [NullAllowed] NSObject target, [NullAllowed] Selector action);
+
+		[Static]
+		[Export ("comboButtonWithTitle:image:menu:target:action:")]
+		NSComboButton CreateWithTitle (string title, NSImage image, [NullAllowed] NSMenu menu, [NullAllowed] NSObject target, [NullAllowed] Selector action);
+
+		[Export ("title")]
+		string Title { get; set; }
+
+		[NullAllowed, Export ("image", ArgumentSemantic.Strong)]
+		NSImage Image { get; set; }
+
+		[Export ("imageScaling", ArgumentSemantic.Assign)]
+		NSImageScaling ImageScaling { get; set; }
+
+		[Export ("menu", ArgumentSemantic.Strong)]
+		NSMenu Menu { get; set; }
+
+		[Export ("style", ArgumentSemantic.Assign)]
+		NSComboButtonStyle Style { get; set; }
 	}
+
+	interface INSPreviewRepresentableActivityItem { }
+
+	[NoMacCatalyst, Mac (13,0)]
+	[Protocol]
+	[BaseType (typeof(NSObject))]
```

Comment:
This needs to be removed or the ModelAttribute, I think the former is the right choice 
```suggestion
```